src/pytorch_lightning/strategies/ddp.py

```diff
@@ -72,6 +71,10 @@
 if _TORCH_GREATER_EQUAL_1_10 and torch.distributed.is_available():
     from torch.distributed.algorithms.model_averaging.averagers import ModelAverager
 
+if torch.distributed.is_available():
```

Comment:
@Borda a bug in the wrapper generation is blocking this PR: https://github.com/Lightning-AI/lightning/runs/8136720509?check_suite_focus=true

```python
src/lightning/pytorch/strategies/fully_sharded_native.py:3: [F821] undefined name 'torch'
src/lightning/pytorch/strategies/ddp.py:3: [F821] undefined name 'torch'
```